app/code/Magento/Backend/Controller/Adminhtml/Cache/MassDisable.php

```diff
@@ -60,12 +60,17 @@ private function disableCache()
             }
             if ($updatedTypes > 0) {
                 $this->_cacheState->persist();
-                $this->messageManager->addSuccess(__("%1 cache type(s) disabled.", $updatedTypes));
+                $this->messageManager->addSuccessMessage(__("%1 cache type(s) disabled.", $updatedTypes));
             }
         } catch (LocalizedException $e) {
-            $this->messageManager->addError($e->getMessage());
+            $this->messageManager->addComplexErrorMessage(
+                'addUnescapedMessage',
```

Comment:
@adrian-martinez-interactiv4 Where is `addUnescapedMessage` message renderer defined? I can not find it in `2.2-develop` or this PR. Before use of complex message it should be registered in `\Magento\Framework\View\Element\Message\MessageConfigurationsPool` (e.g. https://github.com/magento/magento2/blob/2.2-develop/app/code/Magento/UrlRewrite/etc/adminhtml/di.xml#L19)